src/kata6.joinNames.js

```diff
@@ -1,3 +1,17 @@
-const joinNames = (namesObj) => {};
+/* eslint-disable no-plusplus */
+const joinNames = (namesObj) => {
+    const namesArr = namesObj.map(person => person.name);
```

Comment:
The indentation is off in this file, everywhere else you have 2 spaces but in here you have 4. This should be make the same across the whole project